scala/src/com/google/idea/blaze/scala/run/BlazeScalaTestEventHandler.java

```diff
@@ -0,0 +1,37 @@
+package com.google.idea.blaze.scala.run;
+
+import com.google.idea.blaze.base.model.primitives.Kind;
+import com.google.idea.blaze.base.run.smrunner.BlazeTestEventsHandler;
+import com.google.idea.blaze.base.run.smrunner.SmRunnerUtils;
+import com.intellij.execution.Location;
+import com.intellij.execution.testframework.JavaTestLocator;
+import com.intellij.execution.testframework.sm.runner.SMTestLocator;
+import com.intellij.openapi.project.Project;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class BlazeScalaTestEventHandler implements BlazeTestEventsHandler {
+    @Override
+    public boolean handlesKind(@Nullable Kind kind) {
+        return kind == Kind.SCALA_JUNIT_TEST;
+    }
+
+    @Nullable
+    @Override
+    public SMTestLocator getTestLocator() {
+        return JavaTestLocator.INSTANCE;
+    }
+
+    @Nullable
+    @Override
+    public String getTestFilter(Project project, List<Location<?>> testLocations) {
+        return null;
+    }
+
+    @Override
+    public String testDisplayName(@Nullable Kind kind, String rawName) {
+        int index = rawName.lastIndexOf(SmRunnerUtils.TEST_NAME_PARTS_SPLITTER);
```

Comment:
This converts `foo should::bar` to just `bar` right?
I think we want to keep the whole line except replace `::` with a space, since only the second part doesn't make sense when read as a sentence.

E.g., `this function should return this value` instead of just `return this value`.

Reply:
There was some issue that the name was displayed as `::bar`, for some reason. I haven't investigated yet if this was a problem with the xml. This was a quick fix to strip the `::` character from the test name. They are still show up under "should", see image:
![nov-07-2017 10-39-26](https://user-images.githubusercontent.com/601206/32690885-233243b4-c707-11e7-9fa8-7ba2ff801531.gif)

before this change, the name would show as `::print hello`.